app/views/epp/poll/_action.xml.builder

```diff
@@ -1,9 +1,10 @@
 builder.extension do
   builder.tag!('changePoll:changeData',
-               'xmlns:changePoll' => Xsd::Schema.filename(for_prefix: 'changePoll')) do
+               'xmlns:changePoll' => Xsd::Schema.filename(for_prefix: 'changePoll', for_version: '1.0')) do
```

Comment:
In some files, including this one, it is not quite in terms of clean code declaration code. Since with each schema change we need to change the schema version as well, it can take some time to refactor code like this.

For example, if you look in the `app/views/epp/domain/create.xml.builder` file, there is a class variable in the versions in the place where the version and prefix are specified. I suggest you do the same here

Reply:
in case of domain requests it is possible to parse schema prefix and version, domain_controller uses method parse_schemas_prefix_and_version and assigns: @schema_version = res[:version]
@schema_prefix = res[:prefix], which then are used in views. In poll requests there is no changePoll version.

Reply:
Theoretically, a version may appear in the future. In any case, hardcoding versions in plain text is not considered the best approach. There must be at least a constant in the file that specifies the schema and prefix versions. It was the same with domains like you have now. There were no new schemas, but then the prefix and version changed and I had to go through a large number of methods, classes and tests and change versions and prefixes everywhere. This needs to be done so that if in the future it will be necessary to change something, so that it does not take much time.

Since your method accepts a prefix and a version, it is obvious that these are dependent things and they are taken out as a separate constant or raised up in the parent class.